src/modules/file-share/stop-file-share/stop-file-share.js

```diff
@@ -0,0 +1,52 @@
+(function() {
+    "use strict";
+
+    var app = angular.module("TendrlModule");
+
+    app.controller("StopFileShareController", StopFileShareController);
+
+    /*@ngInject*/
+    function StopFileShareController($rootScope, $scope, $state, selectedFileShare, fileShareStore) {
```

Comment:
@cloudbehl Its better to change all Fileshare text to Volume at once only. As on some screens it will be volume and on some it will be File Shares, which will not look good. Need to send a separate PR for File Share --> Volume change.